test/integration/targets/azure_rm_webapp_facts/aliases

```diff
@@ -0,0 +1,3 @@
+cloud/azure
+shippable/azure/group4
```

Comment:
i think it would be better to add webapp facts tests to the main webapp test and reuse webapp that is already created there. that would reduce amount of resources consumed by testing.
i did it with vm facts test.

Reply:
nice. moved to web app tests